docgen/.babelrc

```diff
@@ -0,0 +1,56 @@
+{
+  "env": {
+    "development": {
+      "presets": [
+        [
+          "env",
+          {
+            "targets": {
+              "browsers": [
+                "last 2 versions",
+                "ie >= 9"
+              ]
+            }
+          }
+        ],
+        "react",
+        "stage-2"
+      ],
+      "plugins": [
+        "lodash",
+        "inline-json-import",
```

Comment:
This is the plugin that accidentally gives the whole json object for just the version

Reply:
Yes I removed it in the others `.babelrc`. I kept it in `docgen` because we require the whole `package.json` for the configuration.